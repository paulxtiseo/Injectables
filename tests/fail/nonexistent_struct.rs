use injectables::inject_fields;

#[inject_fields(NonexistentStruct)]
pub struct Test {
    pub name: String,
}

fn main() {}