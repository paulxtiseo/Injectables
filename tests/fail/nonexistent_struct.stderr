error: Cannot inject fields from 'NonexistentStruct' as this type does not exist
 --> tests/fail/nonexistent_struct.rs:3:1
  |
3 | #[inject_fields(NonexistentStruct)]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `inject_fields` (in Nightly builds, run with -Z macro-backtrace for more info)
